Examples/Examples/All Examples/AnimateGeoJSONLine.swift

```diff
@@ -0,0 +1,174 @@
+import UIKit
+import MapboxMaps
+import Turf
+
+
+
+@objc(AnimateGeoJSONLine)
+public class AnimateGeoJSONLine: UIViewController, ExampleProtocol {
+    
+    internal var mapView: MapView!
+    internal let sourceIdentifier = "route-source-identifier"
+    internal var routeLineSource: GeoJSONSource!
+    var currentIndex = 0
+    
+    public var geoJSONLine = (identifier: "routeLine", source: GeoJSONSource())
+    
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView = MapView(with: view.bounds, resourceOptions: resourceOptions())
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        self.view.addSubview(mapView)
+        
+        let centerCoordinate = CLLocationCoordinate2D(latitude: 45.5076, longitude: -122.6736)
+        
+        mapView.cameraManager.setCamera(centerCoordinate: centerCoordinate,
+                                        zoom: 11.0)
+        
+        // Allows the delegate to receive information about map events.
```

Comment:
We don't use a delegate-based callback system here anymore, so this comment is outdated.

```suggestion
        // Wait for the map to load its style before adding data.
```